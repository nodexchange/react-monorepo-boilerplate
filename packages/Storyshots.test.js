import initStoryshots from '@storybook/addon-storyshots';
// the required import from the @storybook/addon-storyshots-puppeteer addon
import { imageSnapshot } from '@storybook/addon-storyshots-puppeteer';
import path from 'path';

// function to customize the snapshot location
const getMatchOptions = () => {
  // Generates a custom path based on the file name and the custom directory.
  const snapshotPath = path.join('packages/storyshots/snapshots');
  return { customSnapshotsDir: snapshotPath };
};

initStoryshots({
  // your own configuration
  test: imageSnapshot({
    // invoke the function above here
    getMatchOptions,
  }),
});
