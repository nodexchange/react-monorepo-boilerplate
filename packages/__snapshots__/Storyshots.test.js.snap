// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Storyshots Boilerplate/Button Default 1`] = `
<a
  className="sc-bdnxRM dibsln"
>
  click me
</a>
`;

exports[`Storyshots Boilerplate/Demo Default 1`] = `
<div>
  demo
</div>
`;
